tiddit/tiddit_cluster.pyx

```diff
@@ -70,7 +70,7 @@ def main(prefix,chromosomes,contig_length,samples,is_mp,epsilon,m,max_ins_len,mi
 				if int(posB) > contig_length[chrB]:
 					posA=contig_length[chrB]
 	
-			discordants[chrA][chrB].append([content[0],sample,"D",posA,content[5],posB,content[8],i])
+			discordants[chrA][chrB].append([content[0],sample,"D",posA,content[5],posB,content[8],i,int(content[3]),int(content[4]),int(content[6]),int(content[7])])
```

Comment:
This could be less than perfectly easy to maintain. Consider naming the content columns.😉